src/modules/liquiditypool.ts

```diff
@@ -178,15 +179,13 @@ export class LiquidityPoolModule extends BaseModule {
   }
 
   public async getWeeklyRewardsRealInflation(): Promise<BigNumber> {
-    const { result }: InsightsQueryResponse<QueryGetInflation> = await this.sdkProvider.insights.Inflation()
-    const MIN_RATE = new BigNumber(0.0003);
-    const INITIAL_SUPPLY = new BigNumber(1000000000);
-    let inflationRate = new BigNumber(result.inflationRate);
-
-    if (inflationRate.lt(MIN_RATE)) {
-      inflationRate = MIN_RATE;
+    const mintDataResponse: Models.QueryMintDataResponse = await this.sdkProvider.query.inflation.MintData({})
+    let weeklyRewards = BN_ZERO
+    if (mintDataResponse) {
```

Comment:
Think `mintDataResponse` is alr declared as `Models.QueryMintDataResponse` so it will always be defined. Think `if (mintDataResponse.mintData)` makes more sense instead.